t/kit-validation.t

```diff
@@ -18,9 +18,11 @@ run_fails "genesis new failures-galore > errors 2>&1", 255, "Invalid kits fail t
 eq_or_diff get_file("errors"), <<EOF, "Invalid kits generate expected error messages";
 Generating new environment failures-galore...
 
-Using dev/ (development version) kit...
+Using local development kit (./dev)...
 
-The following errors have been encountered validating the dev/latest kit:
+\e[1;33mDEPRICATED:\e[0m subkits are depricated in favour of features.  Please update your kit.
```

Comment:
DEPRECATED has 3 E's

Reply:
Can I claim that it's the Canadian spelling?